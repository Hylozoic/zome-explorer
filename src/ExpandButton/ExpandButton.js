import React from 'react'
import { FiChevronRight, FiChevronDown } from 'react-icons/fi'

export default function ExpandButton ({ expanded, toggleExpanded }) {
  return <div className='expand-button' onClick={toggleExpanded}>
    {!expanded && <FiChevronRight className='expand-button-icon' />}
    {expanded && <FiChevronDown className='expand-button-icon' />}
  </div>
}