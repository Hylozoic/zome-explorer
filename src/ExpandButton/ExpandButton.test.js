import ExpandButton from './ExpandButton'
import { shallow } from 'enzyme'
import React from 'react'

it('matches the last snapshot', () => {
  const wrapper = shallow(<ExpandButton toggleExpanded={() => {}} />)
  expect(wrapper).toMatchSnapshot()
})

it('matches the last snapshot when expanded', () => {
  const wrapper = shallow(<ExpandButton toggleExpanded={() => {}} expanded />)
  expect(wrapper).toMatchSnapshot()
})
