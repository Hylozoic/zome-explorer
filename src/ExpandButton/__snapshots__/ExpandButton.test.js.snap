// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the last snapshot 1`] = `
<div
  className="expand-button"
  onClick={[Function]}
>
  <FiChevronRight
    className="expand-button-icon"
  />
</div>
`;

exports[`matches the last snapshot when expanded 1`] = `
<div
  className="expand-button"
  onClick={[Function]}
>
  <FiChevronDown
    className="expand-button-icon"
  />
</div>
`;
